python/ray/tests/test_client_builder.py

```diff
@@ -254,8 +254,10 @@ def test_disconnect(call_ray_stop_only):
         ray.put(300)
 
 
-def test_address_resolution(ray_start_regular_shared):
-    server = ray_client_server.serve("localhost:50055")
+def test_address_resolution(call_ray_stop_only):
+    subprocess.check_output(
+        "ray start --head --ray-client-server-port=50055", shell=True)
```

Comment:
why do we use subprocess instead of `server.server`?

Reply:
Oh to use the proxy!